vision/beta/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -90,14 +90,45 @@ public void testDetectLocalizedObjectsGcs() throws Exception {
   public void testDetectHandwrittenOcrGcs() throws Exception {
     // Act
     String[] args = {
-        "handwritten-ocr",
-        "gs://cloud-samples-data/vision/handwritten.jpg",
+      "handwritten-ocr", "gs://cloud-samples-data/vision/handwritten.jpg",
     };
     Detect.argsHelper(args, out);
 
     // Assert
     String got = bout.toString();
     assertThat(got).contains("Google Cloud Platform");
+  }
+
+  @Test
+  public void testDetectDocumentFeatures() {
+    // Act
+    DetectBatchAnnotateFiles.detectBatchAnnotateFiles("./resources/kafka.pdf");
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("Confidence:");
```

Comment:
Should add a check to verify some of the results returned form the API.

Reply:
Fixed in latest commit.